client/app/account/user/credentials/user-credentials.service.js

```diff
@@ -0,0 +1,40 @@
+angular.module("UserAccount.services")
+    .service("UseraccountCredentialsService", class UseraccountCredentialsService {
```

Comment:
It should be in ovh-api-services :)

Reply:
You're right! I'll have to merge and release this [pull request](https://github.com/ovh-ux/ovh-api-services/pull/84) first.